guides/image-sources/provisioning-installer-httpboot.plantuml

```diff
@@ -0,0 +1,59 @@
+@startuml
+
+!include foreman.pstyle
+
+title Installer-based provisioning with UEFI HTTP boot on bare metal
+
+actor       User
+participant "Provisioned\nHost" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
+participant DHCP
```

Comment:
In the #3304 PR, there is a `TFTP participant`. Should we add the `participant HTTP` here as well?

Reply:
I believe that we discussed last week that we will not introduce an HTTP participant but treat the Foreman Proxy as the server that provides HTTP.